compiler/luci/pass/src/RemoveRedundantTranspose.cpp

```diff
@@ -61,9 +60,10 @@ bool remove_consecutive_transpose_function(luci::CircleNode *node)
   }
   else
   {
-    auto g = main_perm->graph();
-    auto new_const_node = g->nodes()->create<luci::CircleConst>();
+    auto g = main_node->graph();
 
+    // Create New Const Node
```

Comment:
I don't thing adding comments is aboutr `cover more case`. it would be better to post these changes in another PR.

Reply:
Okay I will remove this.